### DISCUSSION TITLE
Go delve debugger "Command failed: Access is denied." when setting breakpoint

### DISCUSSION THREAD

**Discussion by jordaniversen** [go-delve/delve#2955] [General]
Hello,

I'm using the latest Go (v1.18) on Windows 10 Enterprise (v1909) and the latest Delve (v1.8.2) and when I run `dlv debug` from the command prompt to debug the simplest hello world program, and then try to add any breakpoint (`b main.main:1`) , it displays `Command failed: Access is denied.`

I've confirmed the folder and source files are writeable, and I've also run `dlv` from an Administrator command prompt with the same result.  Adding command prompt option `--log` to show verbose logging doesn't seem to provide any further context.  Debugging was working when I originally installed Go and delve but the debugging started giving this error about a week later.

Thank you

**Comment by aarzilli** [comment]
This is likely caused by some antivirus/security software on your computer.

**Reply by jordaniversen** [reply]
Thanks, that was it.  I spent a lot of time searching and couldnt find this answer.  Is there a FAQ or troubleshooting guide that this can be added to?

Thank you
